### PR TITLE
2016item definitions

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"version": "2.0.0",
```
Comment (QuentinGruber) [inline .vscode/tasks.json:2]: wut wasn't this already there

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "@types/node": "16.7.9",
     "debug": "4.3.2",
     "eul2quat": "1.0.0",
+    "fs": "^0.0.1-security",
```
Comment (QuentinGruber) [inline package.json:18]: this should be removed. "fs" is already in nodejs

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

**Review by Z1Meme — COMMENTED** [review]
👍

**Review by Z1Meme — COMMENTED** [review]
👍

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**